README.md

```diff
@@ -57,7 +57,10 @@ the set up is pretty simple. Just run:
 ```
 $ mkvirtualenv ansible
 (ansible)$ git clone https://github.com/dimagi/commcare-cloud.git
+(ansible)$ pip install pip-tools
+(ansible)$ pip-sync commcare-cloud/requirements.txt
 (ansible)$ pip install -e commcare-cloud/
+(ansible)$ pip install -e commcare-cloud/commcare-cloud-bootstrap/
```

Comment:
I missed this in my first pass, but I think it'd be better not to include `commcare-cloud-bootstap` install instructions here.

I think maybe we could mention in a different section that the tool exists and this is how you install it but that it's not intended for widespread use, but it might be just as good to leave out altogether, since it's not documented and isn't an intended use for it beyond what you and I are currently working on (and now Farid).

Reply:
https://github.com/dimagi/commcare-cloud/pull/1944/commits/7bdf83fa5c60741363e49282a9ada79998d64d0e